src/main/scala/outwatch/dom/dsl.scala

```diff
@@ -0,0 +1,24 @@
+package outwatch.dom
+
+object dsl extends Attributes with Tags with Styles {
+  object tags extends Tags {
+    object extra extends TagsExtra
+    object all extends Tags with TagsExtra
```

Comment:
I think `import dsl.tags._, extra._` is good enough for the rare(?) use case of needing extra tags.